### ISSUE TITLE
install problem with tensorflow 2.0

### ISSUE THREAD

**Issue by lacalaca85** [afeinstein20/eleanor#126]
Hi,
Tried to install the code on an older MBP (OSX 10.10.5, python 3.6), but run into version incompatibility issues after the release of tensorflow 2.0:
```Could not find a version that satisfies the requirement tensorflow<=1.14.0 (from eleanor) (from versions: )
No matching distribution found for tensorflow<=1.14.0 (from eleanor)
```
And then: 
```
> pip search tensorflow
tensorflow (2.0.0)                           - TensorFlow is an open source machine
                                               learning framework for everyone.
> pip install tensorflow==1.14.0
Collecting tensorflow==1.14.0
  Could not find a version that satisfies the requirement tensorflow==1.14.0 
  (from versions: )
No matching distribution found for tensorflow==1.14.0
```

**Comment by paulkerry1** [comment]
Following up on this, I was just wondering if you will be supporting tensorflow 2 (v2.1.0 is currently released) as I notice you have pinned "tensorflow<=1.14.0" in setup.py ?

Cheers
Paul.

**Comment by benmontet** [comment]
Yes, this is on my list, but not at an extremely high priority. I have a couple funding proposals coming up to work through first, so the timescale is weeks rather than days.

The only place we use tensorflow is in the PSF modeling routines, and everything else should work fine without the tensorflow dependence.

I have broader goals to rewrite the entire PSF modeling suite with some new knobs to turn and other improvements, and making it tf2.0 compatible will be a part of that.

**Comment by hposborn** [comment]
I'd like to bump this request, as it seems pip wont install any version with tensorflow<2 on python>3.7. So I'm  having to downgrade to python-3.6 specifically to allow Eleanor to install.

**Comment by benmontet** [comment]
Hey all,

I was just able to install tensorflow 1.15.2 without any problems on python 3.7.6. I seem to recall once having issues with tensorflow and numpy, but with 1.18.1 for that package I had no issues. 

It's on the list, but I have no experience with tf2 so I need to learn before I can implement it. There's zero chance this will get done before the ESO proposal deadline but I can put aside some time in the couple weeks after that and if it's easy implement it. If you have any experience and want to take a crack at it, you're also welcome to!

For now, the only functions that use tensorflow are in the psf modeling routines. We even tried to hide all the imports in there, so everything should work totally fine without having tensorflow as long as you don't try to do any psf modeling. So other than that functionality you should be good to go if you just did the hacky thing and removed tensorflow from your setup.py file.
